Rocket.Chat/Managers/PushManager.swift

```diff
@@ -0,0 +1,64 @@
+//
+//  PushManager.swift
+//  Rocket.Chat
+//
+//  Created by Gradler Kim on 2017. 1. 23..
+//  Copyright © 2017 Rocket.Chat. All rights reserved.
+//
+
+import Foundation
+
+class PushManager {
+    
+    static func setPushToken(completion: @escaping MessageCompletion) {
+        guard let deviceToken = getDeviceToken() else { return }
+        var userId: Any?
+        userId = AuthManager.isAuthenticated()?.userId ?? NSNull()
```

Comment:
Actually, you do not have to pass 'userId' at this point. 
It only passes the 'userId' in the receiving parameter. (Referring to Android source)
At this point, only the 'token' and the 'pushId' are sent to the server, and the PushManager.setUser method actually maps them to 'userId' and 'pushId'.

That's why I do not think you need to return even if you do not have a 'userId'.